docs/elements.md

```diff
@@ -731,7 +728,10 @@ The attributes `none-of-the-above` and `all-of-the-above` can be set to one of t
 - `incorrect`: the corresponding choice will always be shown and will always be an incorrect answer (i.e., a distractor).
 - `true`: same as `random`, accepted for backwards compatibility.
```

Comment:
Removed this line in the latest push, since we want people using the non-deprecated attribute names.